assignment4/substractionUsingPromise.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Title</title>
+        <script>
+            function getPromiseToDoSubstraction(number1, number2) {
```

Comment:
No necessary to use `Promise` word in the function name, simply can be `doSubstraction`